src/pages/ScreeningPage.vue

```diff
@@ -1,28 +1,161 @@
 <template>
   <div class="page-wrapper">
-   <ActionBar name="Screenings"/>
-   <ScreeningsHeader sizeType="large"/>
-   <ScreeningsList/>
+    <ActionBar name="Screenings" />
+    <ScreeningsHeader sizeType="large" />
+    <div class="screen-action">
+      <div class="screen-action__days">
+        <DateButton
+          v-for="item in screeningsDays"
+          :caption="item.name"
+          :valueDate="item.date"
+          :key="item.name"
+        />
+
+      </div>
+      <div class="screen-action__select">
+        <label for="select">
+          <span class="select__label">Movie</span>
+          <select
+            name="movies"
+            class="select"
+            v-model="selected"
+            @change="$emit('input', $event.target.value)"
+          >
+            <option value="All">All movies</option>
+            <option
+              v-for="option in options"
+              :key="option"
+              :value="option"
+            >{{option}}</option>
+
+          </select>
+
+        </label>
+      </div>
+    </div>
+    <section class="list-wrapper">
+      <ScreeningCard
+        v-for="movie in filteredByTitle"
+        :key="movie.id"
+        :movie="movie"
+      />
+
+    </section>
   </div>
 </template>
 
 
 <script>
-
-import ScreeningsHeader from '../components/ScreeningsHeader.vue';
-import ScreeningsList from '../components/ScreeningsList.vue';
-import ActionBar from '../components/ActionBar.vue';
+import ScreeningsHeader from "../components/ScreeningsHeader.vue";
+import ScreeningCard from "../components/ScreeningCard.vue";
+import ActionBar from "../components/ActionBar.vue";
+import DateButton from "../components/DateButton.vue";
 export default {
-  name: 'ScreeningPage',
-  components: {  ScreeningsHeader, ScreeningsList, ActionBar },
+  name: "ScreeningPage",
+  components: {
+    ScreeningsHeader,
+    ScreeningCard,
+    ActionBar,
+    DateButton,
+  },
+
+  data() {
+    return {
+      selected: "All",
+      screeningsDays: [],
+    };
+  },
+  computed: {
+    options() {
+      return this.$store.getters.titles;
+    },
+    movies() {
+      return this.$store.state.movies;
+    },
+    screenings() {
+      return this.$store.state.screenings;
+    },
+
+    filteredByTitle() {
+      const filteredTitle = this.movies.filter(
+        (movie) => movie.title === this.selected
```

Comment:
it is a good practice to always compare stuff by `id` if it's possible because it is guaranteed to be unique